cves/CVE-2017-9993.yml

```diff
@@ -348,7 +359,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: 'The main mistake was not using an allowlist when reading from a file.'
```

Comment:
Explain what an allowlist is. Our target audience is just starting with security so they might not know what an allowlist is yet. Just a quick brief summary should do.